data/local.yaml

```diff
@@ -1089,3 +1089,11 @@ S21F:
 S22F:
   description: Time-based milestone marking the completion of the Spring 2022 development
     cycle.
+LDM-GEN3:
+  description: 'Show parity of Butler Gen3 with Butler Gen2 systems.
+   We should be able to run processing pipelines with Gen3 which previously
+   ran on Gen2.
+   Demonstrate the ability to write new pipelines using the framework.
+   Framework should be ready to support Preops processing.'
```

Comment:
Is it possible to clarify what "Preops processing" is, by referencing a pipelines milestone, or a document or requirement?